packages/extension/app/background.js

```diff
@@ -46,6 +49,26 @@ const connect = function (host, path, extraOpts) {
     })
   })
 
+  const listenToOnBeforeHeaders = once(() => {
+    browser.webRequest.onBeforeSendHeaders.addListener((details) => {
+      if (
+        details.parentFrameId !== 0
+        || details.type !== 'sub_frame'
+        || details.url.includes('__cypress')
```

Comment:
In order to support cypress in cypress, we are trying to namespace urls so that we know whether they are going to the parent or child cypress instance. In this scenario I don't know that those urls will necessarily contain __cypress. Itcould be something else.

Reply:
That's good to know. Please let me know what you land on for the namespacing and I'll update this.